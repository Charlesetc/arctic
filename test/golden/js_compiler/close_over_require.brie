require close_over

define main [
  close_over.main ()
]
