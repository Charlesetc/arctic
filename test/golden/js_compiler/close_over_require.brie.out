function fn_1_() {
	var x = 3;
	var f = new_closure().fill_in_arguments(x);
	f.fill_in_arguments().call(fn_3_)
}
function fn_3_(x) {
	x
}
var test/golden/js_compiler/close_over_require.brie_close_over = {main: new_closure().fill_in_arguments()}
function main() {
	test/golden/js_compiler/close_over_require.brie_close_over.main.fill_in_arguments().call(fn_1_)
}